app/whats-new/index.js

```diff
@@ -0,0 +1,21 @@
+// Dependencies
+const router = require('express').Router()
+
+const contentfulService = require('../../common/services/contentful')
+
+// Define routes
+router.get('/', (req, res) => res.redirect('/whats-new/content'))
+router.get('/content', async (req, res) => {
```

Comment:
I wonder if there's actually a benefit of doing this as apposed to having the page on `/whats-new` directly?

Reply:
Yeah, makes more sense to just link to `/whats-new` than to a sub file. If we have to we can extend the routing for this path but I don't see it being needed anytime soon.